src/vs/workbench/contrib/notebook/browser/viewModel/markupCellViewModel.ts

```diff
@@ -143,6 +154,14 @@ export class MarkupCellViewModel extends BaseCellViewModel implements ICellViewM
 			statusBarHeight: 0
 		};
 
+		this._commentOptions = configurationService.getValue<IEditorCommentsOptions>('editor.comments');
```

Comment:
This can be added to the `baseCellViewModel`. Also I wonder how it works with language overrides.